spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
```

Comment:
dòng này nghĩa là gì e ?

Reply:
Phần này đặt cleaner database  mặc định là các transaction. Các transaction rất nhanh và hiêu quả đối với tất cả các test mà chúng thực hiện trong đó có toàn bộ test chạy trong quy trình RSpec (Transaction giúp toàn vẹn dữ liệu, các thay đổi trong cơ sở dữ liệu chỉ được giữ lại khi tất cả các câu lệnh SQL trong transaction đều được thực hiện thành công)